src/compiler/builderState.ts

```diff
@@ -491,9 +495,10 @@ export namespace BuilderState {
     export function getExportedModules(exportedModulesFromDeclarationEmit: ExportedModulesFromDeclarationEmit | undefined) {
         let exportedModules: Set<Path> | undefined;
         exportedModulesFromDeclarationEmit?.forEach(
-            symbol => getReferencedFilesFromImportedModuleSymbol(symbol).forEach(
-                path => (exportedModules ??= new Set()).add(path)
-            )
+            symbol =>

```

Comment:
I dont get why this changed.

Reply:
I'm not sure there's any particular reason besides "that's what dprint does".

Reply:
We can of course check if any of these are bugs or can be improved, but I think much is going to come down to preference...

Reply:
I opened https://github.com/dprint/dprint-plugin-typescript/issues/537